external/openliberty-mp-tck/playlist.xml

```diff
@@ -27,7 +27,7 @@
 		</groups>
 	</test>
 	<test>
-		<testCaseName>openliberty_microprofile_tck_jdk8_j9</testCaseName>
+		<testCaseName>openliberty-mp-tck_test</testCaseName>
```

Comment:
See the 'For cases where there are multiple <test> entries...' section in the description of #3157 for guidance on this, so this would get changed to openliberty_mp_tck_openj9_jdk8.  As with lucene_solr, we may choose to change the directory name to replace the '-' with '_'.